com.riverbankcomputing.PyQt.BaseApp.yaml

```diff
@@ -0,0 +1,25 @@
+app-id: com.riverbankcomputing.PyQt.BaseApp
+branch: 5.15-21.08
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: QtWebEngineProcess
+separate-locales: false
+base: io.qt.qtwebengine.BaseApp
+base-version: 5.15-21.08
+x-base-commit: 18745efdd443a44236ea0486179aca74f9a664310fecdcf42199f7e68a754d0a
+modules:
+  - python-packaging-tools/python-packaging-tools.json
+  - pyqt/pyqt.json
+  - pyqt-webengine/pyqt-webengine.json
```

Comment:
I would personally prefer having everything in the single manifest but it's not a blocker.